src/appState.ts

```diff
@@ -45,6 +45,8 @@ export function getDefaultAppState(): AppState {
     collaborators: new Map(),
     shouldCacheIgnoreZoom: false,
     showShortcutsDialog: false,
+    showZenMode: false,
```

Comment:
Sorry to be pedantic but ZenMode is not something you can show. It's something you can enable. zenModeEnabled would be better.